client_plugin/drivers/shared/shared_driver.go

```diff
@@ -40,49 +40,28 @@ import (
 	"strings"
 )
 
-// volStatus: Datatype for keeping status of a shared volume
-type volStatus string
-
 /* Constants
    version:              Version of the shared plugin driver
-   dockerAPIVersion:     docker engine 1.24 and above support this api version
-   dockerUSocket:        Unix socket on which Docker engine is listening
    internalVolumePrefix: Prefix for the names of internal volumes. These
                          volumes are the actual vmdk backing the shared volumes.
    sambaImageName:       Name of the docker hub image we pull as samba server
    sambaUsername:        Default username for all accessing Samba servers
    sambaPassword:        Default password for accessing all Samba servers
 
-   volStateCreating:     Shared volume is being created. Not ready to be mounted.
-   volStateReady:        Shared volume is ready to be mounted but
-                         no Samba service running right now.
-   volStateMounted:      Samba service already running. Volume mounted
-                         on at least one host VM.
-   volStateMounting:     Volume is being mounted, File sharing service is being
-                         started for this volume.
-   volStateUnmounting:   Volume is being unmounted, File sharing service is being
-                         stopped for this volume.
-   volStateDeleting:     Shared volume is being deleted. Not ready to be mounted.
+   stateIdx:             Index for volume state in key-value pair array
+   gRefIdx:              Index for global refcount in key-value pair array
+   infoIdx:              Index for metdata information in key-value pair array
 */
 const (
 	version              = "vSphere Shared Volume Driver v0.2"
-	dockerAPIVersion     = "v1.24"
-	dockerUSocket        = "unix:///var/run/docker.sock"
 	internalVolumePrefix = "InternalVol"
 	// TODO Replace with our own samba later
-	sambaImageName               = "dperson/samba"
-	sambaUsername                = "root"
-	sambaPassword                = "badpass"
-	volStateCreating   volStatus = "Creating"
-	volStateReady      volStatus = "Ready"
-	volStateMounting   volStatus = "Mounting"
-	volStateMounted    volStatus = "Mounted"
-	volStateError      volStatus = "Error"
-	volStateUnmounting volStatus = "Unmounting"
-	volStateDeleting   volStatus = "Deleting"
-	stateIdx                     = 0
-	GRefIdx                      = 1
-	InfoIdx                      = 2
+	sambaImageName = "dperson/samba"
+	sambaUsername  = "root"
+	sambaPassword  = "badpass"
+	stateIdx       = 0
```

Comment:
These are basically useless. The values are returned in the same order as the keys[] requested in ReadMetaData().

If the caller doesnt stick to stateIDX=0, gRefIdx=1... convention, these wont work.

Should we really introduce them to avoid writing 1-2 integers as array indices? (I know.. I introduced them)